src/base/noscript.scss

```diff
@@ -8,6 +8,12 @@
 	display: block !important;
 }
 
+@media screen {
+	%noscript-screen-display-block-important {
```

Comment:
This is very confusing.....why do you have a `@media` with only a placeholder? I'm not sure this does what you think it does